fews_3di/simulation.py

```diff
@@ -149,25 +149,41 @@ def run(self):
         self.simulation_id, self.simulation_url = self._create_simulation(model_id)
 
         laterals_csv = self.settings.base_dir / "input" / "lateral.csv"
-        laterals = utils.lateral_timeseries(laterals_csv, self.settings)
-        self._add_laterals(laterals)
+        if path.exists(laterals_csv):
```

Comment:
pathlib gebruiken (zoals ook elders in de code wordt gedaan):

```suggestion
        if laterals_csv.exists():
```